pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/AvslagForLiteTrygdetidAP.kt

```diff
@@ -0,0 +1,547 @@
+                }
+
+                showIf(trygdeperioderEOSland.isNotEmpty()) {
+                    paragraph {
+                        text(
+                            Bokmal to "Tabellen nedenfor viser perioder du har bodd og/eller arbeidet i EØS-land. Disse periodene er brukt i vurderingen av retten til alderspensjon.",
+                            Nynorsk to "Tabellen nedanfor viser periodar du har budd og/eller arbeidd i EØS-land. Desse periodane er brukt i vurderinga av retten til alderspensjon.",
+                            English to "The table below shows your national insurance coverage in EEA countries. These periods have been used to assess whether you are eligible for retirement pension."
+                        )
+                    }
+
+                    includePhrase(
+                        OpplysningerBruktIBeregningenTrygdetidTabeller.UtenlandskTrygdetid(
+                            trygdeperioderEOSland
+                        )
+                    )
+
+                }
+
+                showIf(trygdeperioderAvtaleland.isNotEmpty()) {
+                    paragraph {
+                        text(
+                            Bokmal to "Tabellen nedenfor viser perioder du har bodd og/eller arbeidet i land som Norge har trygdeavtale med. Disse periodene er brukt i vurderingen av retten til alderspensjon.",
+                            Nynorsk to "Tabellen nedanfor viser periodar du har budd og/eller arbeidd i land som Noreg har trygdeavtale med. Desse periodane er brukt i vurderinga av retten til alderspensjon.",
+                            English to "The table below shows your national insurance coverage in countries Norway has a social security agreement. These periods have been used to assess whether you are eligible for retirement pension."
+                        )
+                    }
+
+                    includePhrase(
+                        OpplysningerBruktIBeregningenTrygdetidTabeller.UtenlandskTrygdetid(
+                            trygdeperioderAvtaleland
+                        )
+                    )
+                }
+
+            }
+            showIf(avslagsBegrunnelse.isOneOf(UNDER_62)) {
+                paragraph {
+                    text(
+                        Bokmal to "Du har søkt om å ta ut alderspensjon før du fyller 62 år. For å ha rett til alderspensjon må du være 62 år. Derfor har vi avslått søknaden din.",
+                        Nynorsk to "For å ha rett til alderspensjon må du vere 62 år. Du har søkt om å ta ut alderspensjon før du fyller 62 år. Derfor har vi avslått søknaden din.",
+                        English to "In order to be eligible for retirement pension you have to be 62 years. You have applied for retirement pension from a date prior to having turned 62. Therefore, we have declined your application.",
+                    )
+                }
```

Comment:
Her manglar vel avsnittet Vi har ikke vurdert om du oppfyller de andre kravene for å få alderspensjon fra folketrygden. frå samme frase?